src/doc/trpl/error-handling.md

```diff
@@ -223,6 +223,66 @@ let input = io::stdin().read_line()
                        .ok()
                        .expect("Failed to read line");
 ```
+
 `ok()` converts the `IoResult` into an `Option`, and `expect()` does the same
 thing as `unwrap()`, but takes a message. This message is passed along to the
 underlying `panic!`, providing a better error message if the code errors.
+
+# Using `try!`
+
+When writing code that calls many functions that return the `Result` type, the
+error handling can be tedious. The `try!` macro hides some of the boilerplate
+of propagating errors up the call stack.
+
+It replaces this:
+
+```
+use std::old_io::{File, Open, Write, IoError};
```

Comment:
Could this try to use the `std::fs` APIs instead? (now that we have them).

The `open_mode` can probably just be `File::create`